packages/allure-mocha/test/fixtures/specs/attachment.ts

```diff
@@ -1,20 +1,23 @@
 import { ContentType } from "allure-js-commons";
-import { suite, test } from "mocha-typescript";
-import { allure } from "../../../runtime";
```

Comment:
Can we keep this import instead of calling `MochaAllureReporter.allureContainer.getImplementation()`.

I see that you export `allureContainer`, so should be possible, and the diff below will be smaller

Reply:
Yes, will change it to the old approach.